ocp_resources/cdi.py

```diff
@@ -1,13 +1,13 @@
-from ocp_resources.resource import NamespacedResource
+from ocp_resources.resource import Resource
 
 
-class CDI(NamespacedResource):
+class CDI(Resource):
     """
     CDI object.
     """
 
-    api_group = NamespacedResource.ApiGroup.CDI_KUBEVIRT_IO
+    api_group = Resource.ApiGroup.CDI_KUBEVIRT_IO
```

Comment:
This should be updated across the wrapper, there are additional places this `ApiGroup` is referenced as a `NamespacedResource`

Reply:
@sarahbx what do you mean?
each resource call his parent.

Reply:
It does due to inheritance, but this is confusing for cluster scoped resources, using both to refer to a non-namespace scoped resource.